superset/db_engine_specs/base.py

```diff
@@ -45,6 +45,7 @@
 from flask import current_app, g, url_for
 from flask_appbuilder.security.sqla.models import User
 from flask_babel import gettext as __, lazy_gettext as _
+from jsonpath_ng import parse
```

Comment:
[optional] `jsonpath_ng.parse` could be wrapped in `utils/json.py`, maybe something like `def redact_sensitive(json, sensitive_paths)`

Reply:
Another good point! I've had issues with JSONPath libraries in the past, wrapping them in a helper method will help in the future.